### PR TITLE
Greeting Message with Actions

### PR DESCRIPTION
**Pull request by violetcraze**
Setting up messages for first issue and pull request. Open to suggestions and better wording. Could alternatively include a message in the pull request and issue templates.

### CODE REVIEW

File: .github/workflows/greetings.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Thank you for contributing and using this repository while learning GitHub. We have fallen behind on new issues and are resolving them in the order they were submitted. It maybe a number of weeks before we get to this one. Sorry for any inconvence this causes.'
```
Comment (shiffman) [inline .github/workflows/greetings.yml]:
```suggestion
        issue-message: 'Thank you for contributing and using this repository while learning GitHub. We have fallen behind on new issues and are resolving them in the order they were submitted. It maybe a number of weeks before we get to this one. Sorry for any inconvenience this causes. Choo choo! 🚂🌈❤️'
```

File: .github/workflows/greetings.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        pr-message: 'Thank you for contributing and using this repository while learning GitHub. We have fallen behind on new pull requests and are resolving them in the order they were submitted. It maybe a number of weeks before we get to this one. Sorry for any inconvence this causes.'
```
Comment (shiffman) [inline .github/workflows/greetings.yml]:
```suggestion
        pr-message:  'Thank you for contributing and using this repository while learning GitHub. We have fallen behind on new issues and are resolving them in the order they were submitted. It maybe a number of weeks before we get to this one. Sorry for any inconvenience this causes. Choo choo! 🚂🌈❤️'
```

Comment (violetcraze) [reply .github/workflows/greetings.yml]: The pr-message is for new pull requests. Did you mean to change it to say issue? I think pull request would make more sense would resolve be the best working for pull requests. Maybe we should go with accepting or merging.

Comment (shiffman) [reply .github/workflows/greetings.yml]: Yes, that was my mistake, thanks for catching!

Comment (shiffman) [reply .github/workflows/greetings.yml]: Fixed and used the term "reviewing" which I think works well here?

**Pull request merged**